ext/fetch/23_response.js

```diff
@@ -263,7 +265,7 @@ class Response {
     const baseURL = getLocationHref();
     const parsedURL = new URL(url, baseURL);
     if (!redirectStatus(status)) {
-      throw new RangeError("Invalid redirect status code.");
+      throw new RangeError(`Invalid redirect status code: received ${status}`);
```

Comment:
I feel like we can drop the "received" here

```suggestion
      throw new RangeError(`Invalid redirect status code: ${status}`);
```